resume.json

```diff
@@ -1,201 +1,185 @@
+        "label": "DevOps Professional",
+        "picture": "",
+        "email": "jason.swindle@gmail.com",
+        "phone": "+1 (256) 275 - 4228",
+        "website": "http://resume.swindle.me",
+        "summary": "",
+        "location": {
+            "address": "",
+            "postalCode": "98109",
+            "city": "Seattle",
+            "countryCode": "US",
+            "region": "Washington"
+        },
+        "profiles": [{
+                "network": "LinkedIn",
+                "username": "JasonSwindle",
+                "url": "https://www.linkedin.com/in/JasonSwindle"
+            },
+            {
+                "network": "Github",
+                "username": "JasonSwindle",
+                "url": "https://github.com/JasonSwindle"
+            },
+            {
+                "network": "Gitlab",
+                "username": "JasonSwindle",
+                "url": "https://gitlab.com/u/JasonSwindle"
+            },
+            {
+                "network": "Keybase",
+                "username": "JasonSwindle",
+                "url": "https://keybase.io/JasonSwindle"
+            }
+        ]
     },
-    "profiles": [
-      {
-        "network": "LinkedIn",
-        "username": "JasonSwindle",
-        "url": "https://www.linkedin.com/in/JasonSwindle"
-      },
-      {
-        "network": "Github",
-        "username": "JasonSwindle",
-        "url": "https://github.com/JasonSwindle"
-      },
-      {
-        "network": "Gitlab",
-        "username": "JasonSwindle",
-        "url": "https://gitlab.com/u/JasonSwindle"
-      },
-      {
-        "network": "Keybase",
-        "username": "JasonSwindle",
-        "url": "https://keybase.io/JasonSwindle"
-      }
+    "work": [{
+            "company": "Amazon Web Services",
+            "position": "Cloud Support Engineer II (Level 5)",
+            "website": "https://aws.amazon.com/",
+            "startDate": "2016-10-01",
+            "summary": "",
+            "highlights": [
+                "Was the only AWS Premium Support Engineer to represent AWS at DockerCon 2016 (Seattle) in 1-on-1 interactions with customers to understand their needs / gather feedback on ECS or AWS.",
+                "Including job functions from Amazon Web Services, Cloud Support Engineer I (Level 4).",
+                "Gave a presentation at the ECS Global SME Summit that had Solution Architects, Product Leadership, and Premium Support.",
+                "Served as the highest tier of support to mitigate escalations to the respective service team, and improve turn around time on customer issues.",
+                "Point of Contact for ECS / ECR / Fargate / EKS / Docker in support for the service teams, Solution Architects, and leadership.",
+                "The Preferred Engineer and single point of contact for all things ECS / ECR / Docker for a large customer that successfully launched a feature that is possibly used daily by 100 million plus users.",
```

Comment:
Change "successfully launched a feature that is possibly used daily by 100 million plus users."

To "successfully launched a feature used by up to 100 million users daily."